app/src/test/java/org/simple/clinic/phone/MaskedPhoneCallerImplementationTest.kt

```diff
@@ -34,6 +37,18 @@ class MaskedPhoneCallerImplementationTest {
     assertThat(caller.calledNumber).isEqualTo(plainNumber)
   }
 
+  @Test
```

Comment:
We should also add a test that the masked phone number is not used when the flag is unset.

Reply:
This test is already there as the first test.